addons/medical_feedback/functions/fnc_playInjuredSound.sqf

```diff
@@ -28,6 +28,15 @@ if (!local _unit) exitWith {
 };
 if !(_unit call EFUNC(common,isAwake)) exitWith {};
 
+// Limit network traffic by only sending the event to players who can potentially hear it
+private _distance = if (_type == "hit") then {
+    [50, 60, 70] select _severity;
+} else {
+    [10, 15, 20] select _severity;
+};
+private _targets = _unit nearEntities ["CAManBase", _distance];
+if (_targets findIf {isPlayer _x} == -1) exitWith {};
```

Comment:
Does not recognize vehicle crew.

Reply:
So the 
```sqf
private _targets = allPlayers inAreaArray [getPosWorld _unit, _distance, _distance]
```
is the solution ?

If any player is inside the area then the `_targets` will not be empty. This also handle the case where player is inside a vehicle or `_unit` is a player.

Reply:
That sounds better yes. But I'm not sure if this actually improved performance

Reply:
56 units, 24 in-range
```
allUnits select {(_x distance player) < 100};    0.0657 ms
allUnits inAreaArray [getPos player, 100, 100];  0.0085 ms